stan-ballerina/annotation.bal

```diff
@@ -17,7 +17,7 @@
 # The configurations for the NATS streaming subscription.
 #
 # + subject - The name of the subject
-# + queueGroup - The name of the queue group
```

Comment:
This is an API change. We're strictly not doing API changes. You will have to revert this. And fix whatever it is from the java side.